src/test/scala/org/kibanaLoadTest/scenario/Dashboard.scala

```diff
@@ -3,132 +3,173 @@ package org.kibanaLoadTest.scenario
-              ).findAll
-                .transform(
-                  _.map(_.replaceAll("\"name(.+?),", ""))
-                ) //remove name attribute
-                .saveAs("searchAndMapVector")
-            )
             .check(status.is(200))
-        )
-        .exec(session =>
-          //convert Vector -> String
+        ).exec(session =>
+          //convert Vector -> String for visualizations request
           session.set(
             "vizListString",
             session("vizVector").as[Seq[String]].mkString(",")
           )
-        )
-        .exec(session => {
-          //convert Vector -> String
+        ).exec(session =>
+          //convert Vector -> String for search&map request
           session.set(
             "searchAndMapString",
             session("searchAndMapVector").as[Seq[String]].mkString(",")
           )
-        })
-        .exec(
+        ).exec(
           http("query visualizations")
             .post("/api/saved_objects/_bulk_get")
-            .body(
-              StringBody(
-                "[" +
-                  "${vizListString}"
-                    .concat(
-                      ", { \"id\":\"${indexPatternId}\", \"type\":\"index-pattern\"  }]"
-                    )
-              )
-            )
+            .body(StringBody("[${vizListString}, {\"id\":\"${indexPatternId}\", \"type\":\"index-pattern\"}]"))
             .asJson
             .headers(headers)
             .header("Referer", baseUrl + "/app/dashboards")
             .check(status.is(200))
-        )
-        .exec(
+        ).exec(
           http("query search & map")
             .post("/api/saved_objects/_bulk_get")
             .body(StringBody("""[ ${searchAndMapString} ]""".stripMargin))
             .asJson
             .headers(headers)
             .header("Referer", baseUrl + "/app/dashboards")
             .check(status.is(200))
-        )
-        .exec(
+        ).exec(
+          http("query index pattern meta fields")
+          .get("/api/index_patterns/_fields_for_wildcard")
+            .queryParam("pattern", "kibana_sample_data_ecommerce")
+            .queryParam("meta_fields", "_source")
+            .queryParam("meta_fields", "_id")
+            .queryParam("meta_fields", "_type")
+            .queryParam("meta_fields", "_index")
+            .queryParam("meta_fields", "_score")
+            .headers(headers)
+            .header("Referer", baseUrl + "/app/dashboards")
+            .check(status.is(200))
+        ).exec(
+          http("query index pattern search fields")
+            .get("/api/saved_objects/_find")
+            .queryParam("fields", "title")
+            .queryParam("per_page", "10")
+            .queryParam("search", "kibana_sample_data_ecommerce")
+            .queryParam("search_fields", "title")
+            .queryParam("type", "index-pattern")
+            .headers(headers)
+            .header("Referer", baseUrl + "/app/dashboards")
+            .check(status.is(200))
```

Comment:
I saw this call only a few times and so far on cloud only, what are the conditions for it to be made?

Reply:
@mattkime do you have some context here?

Reply:
Heh, I have zero context